src/pages/Causes/SignupForm.vue

```diff
@@ -167,7 +167,13 @@ export default {
 		completeCausesBraintree() {
 			// disable form inputs while routing to thanks
 			this.success = true;
+			// fire events
 			this.$kvTrackEvent('Registration', 'successful-causes-reg', 'register-causes');
+			// Track facebook event
+			if (typeof window !== 'undefined' && typeof fbq === 'function') {
+				window.fbq('track', 'CompleteRegistration');
```

Comment:
I think `CompleteRegistration` should have a signifier that points to "Causes", maybe "CompleteCauseRegistration" so that we don't confuse this with an account Registration.